commands/info.md

```diff
@@ -172,6 +172,7 @@ by referring to the `MEMORY STATS` command and the `MEMORY DOCTOR`.
 Here is the meaning of all fields in the **persistence** section:
 
 *   `loading`: Flag indicating if the load of a dump file is on-going
+*   `async_loading`: To differentiate from the blocking loading - Added in Redis 7.0.
```

Comment:
```suggestion
*   `async_loading`: Currently busy loading replication data-set asynchronously while serving old data. This means `repl-diskless-load` is enabled and set to `swapdb`. Added in Redis 7.0.
```

Reply:
LTGM, one suggestion: maybe drop the word "busy", (not to be confused with `-BUSY` error).
the text "currently loading..." would be enough..